src/java/org/apache/cassandra/service/paxos/PrepareVerbHandler.java

```diff
@@ -45,12 +46,21 @@ public void doVerb(Message<Commit> message)
         // Prepare phase incorporates a read to check the cas condition, so a read sensor is registered in addition to the write sensor
         sensors.registerSensor(context, Type.READ_BYTES);
         sensors.registerSensor(context, Type.WRITE_BYTES);
+        sensors.registerSensor(context, Type.INTERNODE_BYTES);
+        sensors.incrementSensor(context, Type.INTERNODE_BYTES, message.payloadSize(MessagingService.current_version));
         ExecutorLocals locals = ExecutorLocals.create(sensors);
         ExecutorLocals.set(locals);
 
         Message.Builder<PrepareResponse> reply = message.responseWithBuilder(doPrepare(message.payload));
+
         SensorsCustomParams.addWriteSensorToResponse(reply, sensors, context);
         SensorsCustomParams.addReadSensorToResponse(reply, sensors, context);
+
+        // calculate outbound internode bytes before adding the sensor to the response
```

Comment:
I think there is no need to do this since sensor bytes are stored in the header and we only use payload size.

Reply:
The idea behind `INTERNODE_BYTES` is to capture the size of the payload for both inbound and outbound message. line 60 is doing the outbound calculation.

Reply:
I see. I was thinking to minimize number of calls to `currentPayloadSize` since it's on the hot code path and builds a message to get the size...